src/backend.f90

```diff
@@ -147,6 +182,10 @@ subroutine alloc_tdsops(self, tdsops, n, dx, operation, scheme)
          integer, intent(in) :: n
          real(dp), intent(in) :: dx
          character(*), intent(in) :: operation, scheme
+         integer, optional, intent(in) :: n_halo
+         character(*), optional, intent(in) :: from_to, bc_start, bc_end
+         logical, optional, intent(in) :: sym
+         real(dp), optional, intent(in) :: c_nu, nu0_nu
```

Comment:
Could you explain what these new arguments are for?
Also, `alloc_tdsops` hasn't been updated to use these arguments, maybe mention it is a WIP?

Reply:
Thanks for pointing out, alloc_tdsops methods in backends were not passing all these optional arguments down to tdsops_init but now it is all fixed.